tests/test_cms_menus.py

```diff
@@ -839,6 +836,91 @@ def test_menu_with_softroot_page_rendering(self):
 
         self.assertTreeQuality(soft_root, mock_tree, 'level', 'title')
 
+    def test_menu_with_hidden_softroot_page_rendering(self):
+        """
+        Tree in fixture :
+               root
+                   aaa (soft_root)
+                       aaa1
+                           ccc
+                               ddd
+                       aaa2
+                   bbb
+        tag: show_menu 0 100 100 100
+        expected result:
+            renders just the softroot page, with no menu nodes
```

Comment:
We're still missing the why, if this broke I would have no idea what I'd actually broke, jus that this test is failing for some reason. I suspect it renders no nodes because the soft_root and hide_node are True and the requirement is that when rendering a node that is soft_root and hide_node the navigation should be empty?

If the above is correct we also need to edit the above to: aa (soft_root, hide_node)

Reply:
yes you are right when rendering node that is soft_root and hide_node Navigation is empty.